.github/workflows/pilotWrapper.yml

```diff
@@ -11,10 +11,10 @@ jobs:
       fail-fast: False
       matrix:
         python:
-          - 2.6.9
```

Comment:
isn't the pilot still supposed to be able to run on 2.6 ?

Reply:
Well, 2.6 was for SLC6, which I believe that by now is finally gone (we hopefully won't find anymore Worker Nodes with SLC6, and even more by the time this go to production).

Reply:
I wouldn't bet on that, but okay :-)